mysql/infoschema.go

```diff
@@ -28,8 +28,8 @@ import (
 // ProcessInfoSchema performs schema conversion for source database
 // 'db'. Information schema tables are a broadly supported ANSI standard,
 // and we use them to obtain source database's schema information.
-func ProcessInfoSchema(conv *internal.Conv, db *sql.DB) error {
-	tables, err := getTables(db)
+func ProcessInfoSchema(conv *internal.Conv, db *sql.DB, tableSchema string) error {
```

Comment:
`tableSchema` is a very specific term related to `information_schema`. When I first look at this variable name, I have no idea what it should be. I think `dbName` is more clear under `mysql` context. This is just my personal opinion.

Reply:
Yes, I agree: that would be easier to follow.
Maybe even call getTables's parameter dbName, and then have the comment about what is going on within getTables, where we get to make the connection between dbName and MySQL schema.

Note that sql.DB already effectively has the dbName embedded with it (dbName is part of the stuff passed to Open), but unfortunately there is now way to extract it from sql.DB.

Reply:
I have updated parameter to `dbName` and have added a function comment in `getTables()` function.